assignment/tool/src/java/org/sakaiproject/assignment/tool/AssignmentAction.java

```diff
@@ -14975,57 +15121,6 @@ public void doCancel_options(RunData data, Context context) {
         state.setAttribute(STATE_MODE, MODE_LIST_ASSIGNMENTS);
     } // doCancel_options
 
-    /**
```

Comment:
Is this a cleanup or this logic has been moved?

Reply:
Cleanup, it's code unused for 15 years... I'm putting it back so it can stay for 15 more

Reply:
Hahahaha, you're totally right, but let's keep it outside this work.

https://sakaiproject.atlassian.net/browse/SAK-48906

Reply:
I like this @bgarciaentornos , I'll invite to this one:

https://github.com/sakaiproject/sakai/pull/11574/files